src/site/_data/i18n/browser_compat.yml

```diff
@@ -42,3 +42,9 @@ preview:
   pt: Visualizar
   ru: Предварительный просмотр
   zh: 预览
+
+flag:
```

Comment:
I don't know what the process is for requesting a translation, so for now, localizations will have to fall back to the English.